<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Checkit!</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0" colspan="3">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" rowspan="2" colspan="2">
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Add File</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Check It!</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Store</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Add Directory</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="overwriteCheckbox">
        <property name="text">
         <string>Overwrite CRC</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0" colspan="6">
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>784</width>
         <height>366</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListWidget" name="listWidget">
          <property name="contextMenuPolicy">
           <enum>Qt::DefaultContextMenu</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="uniformItemSizes">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCheckit">
    <property name="title">
     <string>Checkit</string>
    </property>
    <addaction name="actionAdd_File_2"/>
    <addaction name="actionAdd_Directory_2"/>
    <addaction name="actionCheck"/>
    <addaction name="actionStore"/>
    <addaction name="actionClear"/>
    <addaction name="separator"/>
    <addaction name="actionQuit_2"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionManual"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuCheckit"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAdd_File">
   <property name="text">
    <string>Add File</string>
   </property>
  </action>
  <action name="actionAdd_Directory">
   <property name="text">
    <string>Add Directory</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit_2">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAdd_File_2">
   <property name="text">
    <string>Add File</string>
   </property>
  </action>
  <action name="actionAdd_Directory_2">
   <property name="text">
    <string>Add Directory</string>
   </property>
  </action>
  <action name="actionCheck">
   <property name="text">
    <string>Check</string>
   </property>
  </action>
  <action name="actionStore">
   <property name="text">
    <string>Store</string>
   </property>
  </action>
  <action name="actionOverwrite_CRC">
   <property name="text">
    <string>Overwrite CRC</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>addDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>check()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>store()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>644</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clearList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>585</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>quit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>743</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addFile()</slot>
  <slot>addDirectory()</slot>
  <slot>check()</slot>
  <slot>store()</slot>
  <slot>clearList()</slot>
  <slot>quit()</slot>
 </slots>
</ui>
